src/main/java/ua/edu/internship/interview/service/dto/user/question/UserQuestionCreateDto.java

```diff
@@ -0,0 +1,9 @@
+package ua.edu.internship.interview.service.dto.user.question;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper=true)
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '=' is not preceded with whitespace.
<!-- __reviewdog__:Cg9kZDEyMmQ1NGRlYTdhOWYSCmNoZWNrc3R5bGU= -->